sound/soc/intel/common/soc-acpi-intel-adl-match.c

```diff
@@ -485,6 +485,14 @@ struct snd_soc_acpi_mach snd_soc_acpi_intel_adl_machines[] = {
 		.drv_name = "adl_cs35l41",
 		.sof_tplg_filename = "sof-adl-cs35l41.tplg",
 	},
+	{
+		.id = "ESSX8326",
```

Comment:
no, don't use the ID field but the comp_ids field to match for ES8336 and 8336. see soc-acpi-intel-glk-match.c